src/Analyzers/Core/Analyzers/Analyzers.projitems

```diff
@@ -33,6 +33,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)EnforceOnBuild.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)IDEDiagnosticIds.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)IDEDiagnosticIdToOptionMappingHelper.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)NamespaceSync\AbstractNamespaceSyncDiagnosticAnalyzer.cs" />
```

Comment:
nit: I think `SyncNamespace` would better match the naming pattern for analyzers listed here